packages/dds/sequence/src/test/intervalCollection.fuzz.spec.ts

```diff
@@ -345,20 +345,27 @@ describe("IntervalCollection fuzz testing", () => {
 	createDDSFuzzSuite(model, {
 		validationStrategy: { type: "fixedInterval", interval: 10 },
 		reconnectProbability: 0.1,
+		numberOfClients: 3,
+		clientJoinOptions: {
+			maxNumberOfClients: 6,
+			clientAddProbability: 0.1,
+		},
 		defaultTestCount: 100,
+		// This option can be used to replay a specific seed from its failure file.
+		// replay: 0,
 		saveFailures: { directory: path.join(__dirname, "../../src/test/results") },
 		// Uncomment this line to replay a specific seed:
 		// replay: 0,
 		parseOperations: (serialized: string) => {
-			let operations: Operation[] = JSON.parse(serialized);
-			// Replace this value with some other interval ID to filter replay of the test suite to only include
-			// interval operations with this ID.
-			const filterIntervalId = "00000000-0000-0000-0000-000000000000";
-			if (filterIntervalId) {
-				operations = operations.filter((entry) =>
-					[undefined, filterIntervalId].includes((entry as any).id),
-				);
-			}
+			const operations: Operation[] = JSON.parse(serialized);
```

Comment:
I thought I broke the `replay` feature for a bit due to this code being enabled. It's here to show usefulness of being able to control operation parsing (if you think the eventual consistency bug is due to handling of a particular bit of data, it can ease debugging). But it doesn't need to be active.